docker/compose.yaml

```diff
@@ -40,7 +40,6 @@ services:
       - ./initialization/:/initialization/:ro
     ports:
       - "8545:8545"
-      - "8546:8546"
```

Comment:
Is this just removing the websocket ports? Makes sense if it is, as we don't need them exposed

Reply:
Yes